config.yaml.sample

```
@@ -30,7 +30,7 @@ gcp_vertex:
   #     "universe_domain": "googleapis.com"
   #   }
 
-generative_models:
+# generative_models:
```

```suggestion
generative_models:
```

Let me know how it can be more clear, but only the first `generative_models` entry is commented out (for documentation purposes)

---

There are child dictionaries below this line which will become.. I don't know. Ignored? Perhaps commend out the entire `generative_models` section?

---

Thanks so much for reviewing.

Oh I guess I didn't really have a strong opinion on whether things were commented.
I really was just coming at it as a totally new user and:
1. installing syftr
2. copying sample config file
3. trying to run `syftr --help`

But at step three I see a pydantic validation error - so my thought process was to just have a more "dummy-proof" sample config that at least is valid even if it's not going to get me a usable setup of the library just yet.

Does that make sense?
Also feel free to shoot down this idea entirely if you like.